### PR TITLE
Update Telephony Hook Guide

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Updates Telephony Hooks Guide to better highlight this feature.
    - Highlight ability to use multiple providers
    - Give some possible reasons why orgs might want to do this
    - General wording changes
- **Is this PR related to a Monolith release?** 2024.03.0

### Resolves:

* [OKTA-699446](https://oktainc.atlassian.net/browse/OKTA-699446)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/512a7ec2f6ae90f8f710e5621165109144bf7620/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/83d6a007-9e31-4496-89ad-a057cb7e9f01) | 92  | 75  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.76d6809d-7b58-4883-8ec6-bbf1e6fcd89c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 1]
```diff
@@ -30,39 +30,38 @@ This guide provides an example of an Okta telephony inline hook. This example us
 
 ## Before you begin
 
-* Read the information and perform any required steps in the [Common Hook set-up steps](https://developer.okta.com/docs/guides/common-hook-set-up-steps/nodejs/main/) page first. This includes remixing (copying) the Glitch.com project for this guide and understanding how Glitch projects are configured and used with hooks.
+* Review the information and required steps in the [Common Hook set-up steps](https://developer.okta.com/docs/guides/common-hook-set-up-steps/nodejs/main/) page first. This includes remixing (copying) the Glitch.com project used in this guide and understanding how Glitch projects are configured and used with hooks.
 
 > **Note:** You can also use Amazon Web Services Lambda Serverless Framework as an external service. See [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
 
-* Make sure that you have an active phone number in Twilio with SMS and MMS capabilities.
+* Make sure you have an user in your org with a Phone authenticator enrolled. See [MFA Usage report](https://help.okta.com/okta_help.htm?type=oie&id=ext-mfa-usage).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:37]:
```suggestion
* Make sure you have a user in your org with a Phone authenticator enrolled. See [MFA Usage report](https://help.okta.com/okta_help.htm?type=oie&id=ext-mfa-usage).
```

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 2]
```diff
@@ -30,39 +30,38 @@ This guide provides an example of an Okta telephony inline hook. This example us
...
-
 ## About telephony inline hook implementation
 
-You can customize a telephony Service Provider for your org by using an Okta telephony inline hook. A telephony inline hook allows you to integrate your own custom code into Okta flows that send SMS or Voice (Call) messages (except Okta Verify enrollment). You can integrate this hook with enrollment, authentication, and recovery flows that involve the Phone authenticator. While the one-time passcode (OTP) is sent to the requester, Okta calls your external service to deliver the OTP, and the external service responds with commands that indicate success or failure in delivering the OTP.
+The Okta telephony inline hook allows you to integrate your own custom code into Okta flows that send SMS or voice call messages (except Okta Verify enrollment). You can integrate this hook with enrollment, authentication, and recovery flows that involve phone authenticators. Okta uses your external provider to deliver the one-time passcode (OTP) to the Requester. The provider can respond with commands that indicate if the delivery was successful or not.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:50]:
```suggestion
The Okta telephony inline hook allows you to integrate your own custom code into Okta flows that send SMS or voice call messages (except Okta Verify enrollment). You can integrate this hook with enrollment, authentication, and recovery flows that involve phone authenticators. Okta uses your external provider to deliver the one-time passcode (OTP) to the Requester. The provider can respond with commands that indicate if the delivery was successful.
```

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 3]
```diff
@@ -141,36 +140,36 @@ The following code retrieves the value of the OTP code sent by Okta from the `da
...
+To tell Okta that the SMS or voice call message was successfully sent, return a `commands` object in the body of your HTTPS response. This object is an array that allows you to send multiple commands. The two required properties are `type` and `value`. The`value` property is where you specify the status of your telephony transaction and other relevant transaction metadata. The action type is `com.okta.telephony.action`.
 
 <StackSelector snippet="responsetookta" noSelector/>
 
 ## Preview and test
 
-The external service example is now ready with code to receive and respond to an Okta call. The Okta org is now set up to call the external service using a telephony inline hook. In your Okta org, you can preview the request and response JSON directly from the Admin Console. You can also test the code directly in your org.
+The external service example is now ready with code to receive and respond to an Okta call. The Okta org is now set up to call the external service using a telephony inline hook. In your Okta org, you can preview the JSON-formatted request and response directly from the Admin Console. You can also test the code directly in your org.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:155]:
```suggestion
The external service example is now ready with code to receive and respond to an Okta call. The Okta org is set up to call the external service using a telephony inline hook. In your Okta org, preview the JSON-formatted request and response directly from the Admin Console. You can also test the code directly in your org.
```

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 4]
```diff
@@ -141,36 +140,36 @@ The following code retrieves the value of the OTP code sent by Okta from the `da
...
 
 ### Preview
 
 To preview the telephony inline hook:
 
 1. In the Admin Console, go to **Workflow** > **Inline Hooks**.
-1. Select the telephony inline hook name that you just set up (in this example, **Twilio Telephony Hook**).
+1. Select the telephony inline hook you created (in this example, **Twilio Telephony Hook**).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:162]: avoid past tense
```suggestion
1. Select the telephony inline hook you set up (in this example, **Twilio Telephony Hook**).
```

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 5]
```diff
@@ -141,36 +140,36 @@ The following code retrieves the value of the OTP code sent by Okta from the `da
...
+    > **Note**: If your user hasn't enrolled a phone authenticator, you can manually specify a phone number in the **Preview example inline hook request** section. Click **Edit** and add a value for the `phoneNumber` in the `messageProfile` section of the request (for example, `"+15555551212"`).
 
-1. From the **Preview example inline hook request** section, click **Generate Request**. You should see the user's request information in JSON format that is sent to the external service.
-1. From the **View Service's Response** section, click **View Response**. You should see the response from your external service in JSON format. Upon a successful response, an SMS code or Voice (Call) message with the code is sent to the specified user. If there's an error, the error message appears in the response.
+1. From the **Preview example inline hook request** section, click **Generate Request**. The request sent to the external service is displayed in JSON format.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:169]: displays requires a direct object 
```suggestion
1. From the **Preview example inline hook request** section, click **Generate Request**. The request sent to the external service appears in JSON format.
```

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 6]
```diff
@@ -141,36 +140,36 @@ The following code retrieves the value of the OTP code sent by Okta from the `da
...
+1. From the **View Service's Response** section, click **View Response**. A response from the external service is displayed in JSON format. Upon a successful response, an SMS code or voice call message with the code is sent to the associated user. Upon an error, the error message is returned in the response.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:170]: displays requires a direct object 
```suggestion
1. From the **View Service's Response** section, click **View Response**. A response from the external service appears in JSON format. Upon a successful response, an SMS code or voice call message with the code is sent to the associated user. Upon an error, the error message is returned in the response.
```

**Review by susanharper-okta — COMMENTED** [review]
couple things

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/9667cdedd437e16e80a5048392b8e005bc48eb20/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f6ec217c-dbba-4860-9080-06a824bb9e99) | 92  | 77  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0a42ed24-cc0b-41ce-851a-41d2c9f2bcf0?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/685e3e539a759787b6e7e1abe92aff8a4237b43c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c1c210cf-1b5c-4d23-a603-8e21651b1d5a) | 92  | 77  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.667aa4ee-949c-4d83-91e6-096f31de628f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/927cfe56b530cb100cfb220d7e635c84505a57af/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ea36be2d-befa-4461-ab12-1fd874a6ebaf) | 92  | 77  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9f64d255-5da0-4412-8993-41ed5ee9aa8f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**